### ISSUE TITLE
[Command] /cup results give a response even though there is nothing to show for

### ISSUE THREAD

**Issue by w1lla** [skybaks/pyplanet-cup_manager#41]
If i use a empty database and set the plugin up, i can use /cup matches which gives none which is correct but /cup results shows a window/manialink and i can seem to press each thing.

 ```
[2022-09-16 23:11:34,456][DEBUG][default] peewee.async: ('SELECT `t1`.`id`, `t1`.`created_at`, `t1`.`updated_at`, `t1`.`cup_start_time`, `t1`.`map_start_time` FROM `cup_manager_cupmatch_v1` AS t1 ORDER BY `t1`.`cup_start_time` DESC', ([],), {}) (peewee_async.py:1534)
[2022-09-16 23:11:34,458][DEBUG][default] peewee.async: ('SELECT `t1`.`id`, `t1`.`created_at`, `t1`.`updated_at`, `t1`.`map_start_time`, `t1`.`mode_script`, `t1`.`map_name`, `t1`.`map_uid`, `t1`.`mx_id` FROM `cup_manager_matchinfo_v1` AS t1 ORDER BY `t1`.`map_start_time` DESC', ([],), {}) (peewee_async.py:1534)
[2022-09-16 23:16:09,070][DEBUG][default] peewee.async: ('SELECT `t1`.`id`, `t1`.`created_at`, `t1`.`updated_at`, `t1`.`cup_start_time`, `t1`.`cup_key`, `t1`.`cup_name`, `t1`.`cup_edition`, `t1`.`cup_host_login`, `t1`.`cup_host_nickname` FROM `cup_manager_cupinfo_v1` AS t1 ORDER BY `t1`.`cup_start_time` DESC', ([],), {}) (peewee_async.py:1534)
[2022-09-16 23:16:22,902][DEBUG][default] root: No callback registered for Script.Maniaplanet.EndMap_Start (manager.py:102)
```

Same goes for /cup cups

I know it seems weird behaviour.

**Comment by skybaks** [comment]
Yeah this is a bug which I likely overlooked because my testing database is empty. I'll check into this and correct it.

**Comment by skybaks** [comment]
Fixed as part of #53
